homeassistant/components/whois/sensor.py

```diff
@@ -80,14 +80,21 @@ def _ensure_timezone(timestamp: datetime | None) -> datetime | None:
     return timestamp
 
 
+def _fetch_attr_if_exists(domain: Domain, attr: str) -> str | None:
+    """Fetch an attribute if it exists and is truthy or return None."""
+    if hasattr(domain, attr) and (value := getattr(domain, attr)):
+        return cast(str, value)
+    return None
+
+
 SENSORS: tuple[WhoisSensorEntityDescription, ...] = (
     WhoisSensorEntityDescription(
         key="admin",
         name="Admin",
         icon="mdi:account-star",
         entity_category=EntityCategory.DIAGNOSTIC,
         entity_registry_enabled_default=False,
-        value_fn=lambda domain: domain.admin if domain.admin else None,
+        value_fn=lambda domain: _fetch_attr_if_exists(domain, "admin"),
```

Comment:
Yes.  https://github.com/home-assistant/core/pull/65265

I didn't do it the first time because I assumed it would break the typing, but its loose enough that it works.